src/main/java/frc/robot/Robot.java

```diff
@@ -66,16 +87,42 @@ public void teleopInit() {
       m_autonomousCommand.cancel();
     }
 
+    SwerveDrivetrainSubsystem.getInstance().resetEncoders();
+    
     CommandScheduler.getInstance().setDefaultCommand(
       SwerveDrivetrainSubsystem.getInstance(), 
       new DriveSwerveCommand(
         RobotContainer.driverController::getLeftX,
         RobotContainer.driverController::getLeftY,
         RobotContainer.driverController::getRightX));
+
+    time = Timer.getFPGATimestamp();
+
   }
 
   @Override
-  public void teleopPeriodic() {}
+  public void teleopPeriodic() {
+    double timePassed = Timer.getFPGATimestamp() - time;
+    // start of endgame (20 seconds left)
+    
+
+    // if ((timePassed > 115 && timePassed < 115.2) || (timePassed > 115.4 && timePassed < 115.6) || (timePassed > 115.8 && timePassed < 116)) {
```

Comment:
Don't add so much commented code into main, and in general it's not a good practice to add commented code into git commits, if you might want to use that code again, that's what version control's all about